async/kafka/option.go

```diff
@@ -1,6 +1,7 @@
 package kafka
 
 import (
+	"github.com/beatlabs/patron/encoding"
```

Comment:
Your IDE is not properly setup so it does not eun formating on imports...
Goland->Settinngs->Tools->Filewatchers->add goimports

Reply:
thanks!